### PR TITLE
Move devices code to unit specific directory and update vendor

### PR DESCRIPTION
**Pull request by rhatdan**

### CODE REVIEW

File: pkg/parse/parse_unix.go [hunk 1]
```diff
@@ -19,3 +23,19 @@ func getDefaultProcessLimits() []string {
...
 	return defaultLimits
 }
+
+func DeviceFromPath(device string) (configs.Device, error) {
+	src, dst, permissions, err := Device(device)
+	if err != nil {
+		return configs.Device{}, err
+	}
+	if unshare.IsRootless() {
+		return configs.Device{}, errors.Errorf("Renameing device %s to %s is not a supported in rootless containers", src, dst)
```
Comment (vrothberg) [inline pkg/parse/parse_unix.go]: `s/Renameing/Renaming/` ? It predates this PR but as we're already on it :)

Comment (rhatdan) [reply pkg/parse/parse_unix.go]: Fixed.

**Comment by rhatdan** [comment]
@TomSweeneyRedHat Need to cut a release as soon as this is merged 1.11.2

**Comment by TomSweeneyRedHat** [comment]
LGTM assuming happy tests

**Comment by TomSweeneyRedHat** [comment]
@rh-atomic-bot r+

**Comment by TomSweeneyRedHat** [comment]
This has merged, off to make 1.11.2

**Pull request closed without merging**